block/src/Pos/Block/Network/Logic.hs

```diff
@@ -491,8 +493,13 @@ applyWithoutRollback
     -> OldestFirst NE Block
     -> m ()
 applyWithoutRollback enqueue blocks = do
-    logInfo $ sformat ("Trying to apply blocks w/o rollback: "%listJson) $
-        fmap (view blockHeader) blocks
+    let headers = getOldestFirst $ map (view blockHeader) blocks
+    let fmtString = "Trying to apply blocks w/o rollback." :: Format r r
+    if length headers <= 6
```

Comment:
Probably should put this into a utility function

Reply:
Yes, please at least eliminate copy-paste in this file (move this few lines into a separate function).